js/libs/builder.js

```diff
@@ -77,26 +87,30 @@ Vue.directive('sortable', {
 });
 
 function generateFormDefaults(data) {
-  return _.assign({
```

Comment:
I'm able to open this file and save it without the linter or formatter making any changes ([ref](https://github.com/Fliplet/fliplet-widget-form-builder/blob/5e4483d184debead7ebc47d3f6b8854c1412a4c8/js/libs/builder.js#L79-L100))

![image](https://user-images.githubusercontent.com/290733/228610203-483993ec-b543-4c60-b47a-f4d569ffb1cf.png)

This suggests either you made this change manually or your editor is set up incorrectly.